core/domain/feedback_services.py

```diff
@@ -330,22 +508,32 @@ def _enqueue_feedback_thread_status_change_email_task(
 
 
 def _enqueue_suggestion_email_task(exploration_id, thread_id):
-    """Adds a 'send suggestion email' task into taskqueue."""
+    """Adds a 'send suggestion email' task into task queue.
+
+    Args:
+        exploration_id: str.
+        thread_is: str.
+    """
 
     payload = {
         'exploration_id': exploration_id,
         'thread_id': thread_id
     }
-    # Suggestion emails are sent immidiately.
+    # Suggestion emails are sent immediately.
     taskqueue_services.enqueue_task(
         feconf.TASK_URL_SUGGESTION_EMAILS, payload, 0)
 
 
 def get_feedback_message_references(user_id):
-    """Returns a list of feedback message references corresponding to the given
-    user id. If the user id is invalid or there are no messages for this user,
-    returns an empty list."""
+    """Fetch all FeedbackMessageReference for the given user id
```

Comment:
Seems weird that FeedbackMessageReference sounds singular. Maybe 

```
Fetch all FeedbackMessageReferences for the given user id.
```

or

```
Fetch all FeedbackMessageReference objects for the given user id.
```

That said, I don't actually understand what a "reference for the given user id" means. Does it mean "written by the given user_id"?